src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -24349,58 +24245,7 @@ void Main(string? c, bool b)
         }
 
         [Fact]
-        public void DoesNotReturnIfFalse_Debug_Assert2_NotNull()
```

Comment:
@cston was suggestion that we keep a hard-coded understanding of `Debug.Assert`.
I'm wonder if that's useful. Maybe we could keep those tests, but replace `Assert` with a custom `MyAssert` that is annotated with `DoesNotReturnIf(false)`?

Reply:
Do you believe we have insufficient test coverage for `DoesNotReturnIf(false)`?

---
In reply to: [303224415](https://github.com/dotnet/roslyn/pull/34916#discussion_r303224415) [](ancestors = 303224415)